### PR TITLE
Fixes non cultists being able to use cult bolas

### LINKED ISSUE #57652: Can pickup and use cult bola as non cultist if you have less than 2 legs
## Reproduction:

Lose any number of legs.
Pickup cult bola
use cult bola
win

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## About The Pull Request

Fixes non cultists being able to use cult bolas

Fixes #57652 

## Why It's Good For The Game

non cultists shouldnt be able to use blood magic items

## Changelog
:cl:
fix: Fixes crippled non cultists being able to use cult bola
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
I'm open to lowering/raising the stun for grabbing it to whatever, I just grabbed a value that was already being used for cult gear

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -289,8 +289,14 @@
 /obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/user, list/modifiers)
 	. = ..()
 	if(!iscultist(user))
-		to_chat(user, "<span class='warning'>The bola seems to take on a life of its own!</span>")
-		ensnare(user)
+		if(user.num_legs <= 1)
```
Comment (ATH1909) [inline code/modules/antagonists/cult/cult_items.dm:292]:
```suggestion
		if(C.legcuffed || C.num_legs < 2)
```

**Comment by SuperNovaa41** [comment]
changed the stun to the same time as it takes to breakout of bola for consistency

File: code/modules/antagonists/cult/cult_items.dm [hunk 2]
```diff
@@ -286,11 +286,18 @@
 	breakouttime = 60
 	knockdown = 30
 
-/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/user, list/modifiers)
+/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/carbon/user, list/modifiers)
 	. = ..()
+	var/stun = 6 SECONDS
```
Comment (Mothblocks) [inline code/modules/antagonists/cult/cult_items.dm:291]: Make it a define.

File: code/modules/antagonists/cult/cult_items.dm [hunk 3]
```diff
@@ -286,11 +286,18 @@
 	breakouttime = 60
 	knockdown = 30
 
-/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/user, list/modifiers)
+/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/carbon/user, list/modifiers)
```
Comment (Mothblocks) [inline code/modules/antagonists/cult/cult_items.dm:289]: This can't be guaranteed. Just check `iscarbon`.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/cult/cult_items.dm [hunk 4]
```diff
@@ -286,11 +286,22 @@
...
-/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/user, list/modifiers)
+#define CULT_BOLA_PICKUP_STUN 6 SECONDS
+/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/carbon/user, list/modifiers)
 	. = ..()
+
 	if(!iscultist(user))
-		to_chat(user, "<span class='warning'>The bola seems to take on a life of its own!</span>")
-		ensnare(user)
+		if(iscarbon(user))
+			var/mob/living/carbon/C = user
```
Comment (Mothblocks) [inline code/modules/antagonists/cult/cult_items.dm:295]: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Rename this `carbon_user` or similar.

**Comment by Mothblocks** [comment]
Undraft if ready.

File: code/modules/antagonists/cult/cult_items.dm [hunk 5]
```diff
@@ -286,11 +286,22 @@
...
 
-/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/user, list/modifiers)
+#define CULT_BOLA_PICKUP_STUN 6 SECONDS
+/obj/item/restraints/legcuffs/bola/cult/attack_hand(mob/living/carbon/user, list/modifiers)
 	. = ..()
+
 	if(!iscultist(user))
-		to_chat(user, "<span class='warning'>The bola seems to take on a life of its own!</span>")
-		ensnare(user)
+		if(iscarbon(user))
```
Comment (Mothblocks) [inline code/modules/antagonists/cult/cult_items.dm:294]: Last change--make these one if statement and early return.

i.e.

```dm
if (iscultist(user) || !iscarbon(user))
    return
```

**Pull request merged**